app/models/game.rb

```diff
@@ -1,12 +1,16 @@
 class Game < ActiveRecord::Base
   belongs_to :user
+  belongs_to :sport
   has_many :players, dependent: :destroy
 
   validates :user, presence: true
-  validates :name, presence: true, uniqueness: true
+  validates :name, presence: true,
+    uniqueness: true,
+    length: { maximum: 20}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.